### DISCUSSION TITLE
[v4] How can I display if a query for example a user update was successful or not in a easy way?

### DISCUSSION THREAD

**Discussion by the-prodigy** [adonisjs/core#1736] [Help]
Hey guys,

how can I show in the easiest way if the database query was successful or not when for example a user was deleted or updated, created? 

I have this code here:

```
const user = await User.create(userData)

return response.status(201).json({
            message: "successfully created",
            success: true,
            errors: null,
            data: user
 })
```

But i think it will also appear if the query wasn't successful or not?

Because errors is set to null it will be always be a successful query?

@thetutlage I format my code blocks :o

**Comment by thetutlage** [comment]
There is no need to create 100 threads for a single issue. https://github.com/adonisjs/core/discussions/1737

If query is not successful it will raise an exception. Use the `try/catch` block to catch the exceptions

**Reply by the-prodigy** [reply]
@thetutlage I literally created 2 discussions, so don't be pissed. And thanks, but with try catch its not good code in my way, instead I'm searching a other way.

**Reply by thetutlage** [reply]
There is no need to create even 2 threads. If you are having hard time participating in community, then feel free to leave it.
